Kernel/Devices/HID/VirtIO/EvDevDefinitions.h

```diff
@@ -0,0 +1,988 @@
+/*-
```

Comment:
This file is pretty big (ELFABI.h has a similar line count though), but I don't know if I should strip it down. We don't need all definitions in there, but we might perhaps need some more in the future.

Reply:
Seems like we're accumulating these definition files from third parties :) This one seems to be from FreeBSD though, so meh. Seems fine to just keep the file as-is.

Reply:
I used this FreeBSD header since I wasn't sure how/if I legally could make our own definitions file, since it would use the same numbers (and probably similar names).
I obviously can't use the linux evdev header, since it's GPL.